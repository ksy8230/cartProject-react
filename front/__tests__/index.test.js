import React from 'react'
import { render, fireEvent, getByText } from '@testing-library/react';
import Index from '../pages/index'

it('인덱스 페이지 렌더링', () => {
  const utils = render(<Index />);
  expect(utils.container).toMatchSnapshot();
});