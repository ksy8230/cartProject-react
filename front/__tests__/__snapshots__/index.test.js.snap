// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`인덱스 페이지 렌더링 1`] = `
<div>
  <header>
    <div>
      <a
        href="/"
      >
        홈
      </a>
    </div>
    <div>
      search
    </div>
    <div>
      navagation
    </div>
  </header>
  <a
    href="/products"
  >
    상품리스트
  </a>
</div>
`;
